src/UI/Content/theme.less

```diff
@@ -292,3 +292,9 @@ dl.info {
     vertical-align : bottom;
   }
 }
+
+.sort-direction-icon {
+  .pull-right();
+  position: relative;
+  width: 0px;
```

Comment:
originally the pull-right was in the js file. I can move it back there if you want.